packages/data-extensions/src/surfaces/admin/globals.ts

```diff
@@ -0,0 +1,8 @@
+import type {ExtensionTargets} from './extension-targets';
+
+export interface ShopifyGlobal {
+  dataExtension<ExtensionTarget extends keyof ExtensionTargets>(
```

Comment:
Shouldn't this be `run` to make the shopify global?

Reply:
I'm not following. I thought we want to expose `dataExtension`. Do we call this `run` and then alias it somewhere else?

Reply:
This interface is for defining the Shopify global. You are calling `shopify.run` in this line here: https://github.com/Shopify/ui-extensions/pull/2420/files#diff-d1bf82eda258f81a751d34a89e27f700f4e9b0ab279c63bae4e94f31cc300ea4R15

If this is supposed to be `shopify.dataExtension` then we should update it and also add the global declaration.